src/Umbraco.ModelsBuilder/Umbraco/PureLiveModelFactory.cs

```diff
@@ -24,14 +23,14 @@
 
 namespace Umbraco.ModelsBuilder.Umbraco
```

Comment:
Just noticed this namespace, it's not very pretty and IMO we should change this.

Reply:
Do you have any suggestions?